### ISSUE TITLE
Sources with very long names get cut off on the finder 

### ISSUE THREAD

**Issue by dannygoldstein** [skyportal/skyportal#597]
Example:
<img width="1142" alt="Screen Shot 2020-07-23 at 11 32 05 PM" src="https://user-images.githubusercontent.com/2769632/88366034-d09e7980-cd3c-11ea-8031-6558b98f8a1c.png">

This may prove to be an issue for ZTF, where the objects have very long names.

**Comment by dannygoldstein** [comment]
Here is the corresponding starlist:

```16fil 21 30 52.53 +27 34 26.81 2000.0 #
16fil_off1 21 30 52.32 +27 34 18.41 2000.0 raoffset=2.796 decoffset=8.443 # dist=8.84"; 15.53 mag; 2.796" E, 8.443" N ID=1847293647652953984
16fil_off2 21 30 53.25 +27 33 48.69 2000.0 raoffset=-9.610 decoffset=38.160 # dist=39.32"; 15.33 mag; 9.610" W, 38.160" N ID=1847293613293211776
16fil_off3 21 30 55.30 +27 35 02.38 2000.0 raoffset=-36.776 decoffset=-35.541 # dist=51.19"; 16.05 mag; 36.776" W, 35.541" S ID=1847293750732168960
16f..d_copy 21 30 52.53 +27 34 26.81 2000.0 #
16f..d_copy_off1 21 30 52.32 +27 34 18.41 2000.0 raoffset=2.796 decoffset=8.443 # dist=8.84"; 15.53 mag; 2.796" E, 8.443" N ID=1847293647652953984
16f..d_copy_off2 21 30 53.25 +27 33 48.69 2000.0 raoffset=-9.610 decoffset=38.160 # dist=39.32"; 15.33 mag; 9.610" W, 38.160" N ID=1847293613293211776
16f..d_copy_off3 21 30 55.30 +27 35 02.38 2000.0 raoffset=-36.776 decoffset=-35.541 # dist=51.19"; 16.05 mag; 36.776" W, 35.541" S ID=1847293750732168960
14gqr 23 33 27.95 +33 38 46.14 2000.0 #
14gqr_off1 23 33 27.01 +33 39 19.39 2000.0 raoffset=11.730 decoffset=-33.233 # dist=35.26"; 17.19 mag; 11.730" E, 33.233" S ID=2872657746007579776
14gqr_off2 23 33 27.87 +33 39 21.73 2000.0 raoffset=1.089 decoffset=-35.600 # dist=35.61"; 17.40 mag; 1.089" E, 35.600" S ID=2872657746007579904
14gqr_off3 23 33 26.41 +33 38 08.47 2000.0 raoffset=19.298 decoffset=37.693 # dist=42.33"; 17.23 mag; 19.298" E, 37.693" N ID=2872655516921199744
14g..d_copy 23 33 27.95 +33 38 46.14 2000.0 #
14g..d_copy_off1 23 33 27.01 +33 39 19.39 2000.0 raoffset=11.730 decoffset=-33.233 # dist=35.26"; 17.19 mag; 11.730" E, 33.233" S ID=2872657746007579776
14g..d_copy_off2 23 33 27.87 +33 39 21.73 2000.0 raoffset=1.089 decoffset=-35.600 # dist=35.61"; 17.40 mag; 1.089" E, 35.600" S ID=2872657746007579904
14g..d_copy_off3 23 33 26.41 +33 38 08.47 2000.0 raoffset=19.298 decoffset=37.693 # dist=42.33"; 17.23 mag; 19.298" E, 37.693" N ID=2872655516921199744```

This may prevent users from unambiguously identifying targets based on the star list.

**Comment by stefanv** [comment]
We can fix this on the right side, but in the star list, there is a character limitation:

https://www2.keck.hawaii.edu/observing/starlist.html

**Comment by profjsb** [comment]
@stefanv is right--the limitation is in the starlist formatting for Keck. It maybe that the starlist spec is out of date and that the actual system will accept longer names. Presumably this is a knowable answer.

**Comment by dannygoldstein** [comment]
Fixed by #1084
